releasenotes/notes/iterative-phase-estimation-bugfix-b676ffc23cea8251.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixed a bug where :class:`.IterativePhaseEstimation` was generating the wrong circuit, causing the
+    algorithm to fail for simple cases. Fixed `#9280 https://github.com/Qiskit/qiskit-terra/issues/9280`__.
```

Comment:
There should be a newline at the end of this file. The red circle warns about this.

Reply:
I added a newline but it looks like some tests are still failing. I can't seem to find what exactly is the cause. Do you have any pointers on how to find the error?